### DISCUSSION TITLE
Possible to build output into a custom path?

### DISCUSSION THREAD

**Discussion by germi** [slidevjs/slidev#460] [Q&A]
I have several entry points..:

- slides1.md
- slides2.md
- etc...

and I want to build each one into a separate subfolder, something like...

- /dist
  - /slides1
    - all files...
  - /slides2

I understand there is no unique command to build "all of them at once", but it's fine because I can create a custom script that will do all the builds..:

`npm run build slides1.md`

The problem is that I don't know how to specify the output directory for each one of them. It's always /dist.

I'm trying things like

`npm run build slides1.md --dir=/slides1`

with no luck

Thanks!

**Comment by antfu** [comment]
it's `--output dist`

**Reply by germi** [reply]
Thanks, unfortunately it's not working. This is what I'm trying:

```bash
npm run build slides2.md --output slides2
```

but this is the error I get:

```bash
> build
> slidev build "slides2.md" "slides2"

Unknown argument: slides2
```

**Comment by germi** [comment]
Hello! Just checking if there's anything else I can try... It still doesn't work, I've tried every possible think I could think of...

thanks!!
